tests/interfaces/permissions-grant.spec.ts

```diff
@@ -35,6 +36,56 @@ describe('PermissionsGrant', () => {
       expect(message.descriptor.description).to.eql('drugs');
     });
 
+    describe('scope property normalizations', async () => {
+      it('ensures that `schema` is normalized', async () => {
+        const { privateJwk } = await Secp256k1.generateKeyPair();
+        const authorizationSigner = new PrivateKeySigner({ privateJwk, keyId: 'did:jank:bob' });
+
+        const scope: PermissionScope = {
+          interface : DwnInterfaceName.Records,
+          method    : DwnMethodName.Write,
+          schema    : 'example.com/',
+        };
+
+        const { message } = await PermissionsGrant.create({
+          dateExpires : getCurrentTimeInHighPrecision(),
+          description : 'schema normalization test',
+          grantedBy   : 'did:jank:bob',
+          grantedTo   : 'did:jank:alice',
+          grantedFor  : 'did:jank:bob',
+          scope       : scope,
+          authorizationSigner
+        });
+
+
+        expect((message.descriptor.scope as RecordsPermissionScope).schema).to.equal('http://example.com');
+      });
+
+      it('ensures that `protocol` is normalized', async () => {
+        const { privateJwk } = await Secp256k1.generateKeyPair();
+        const authorizationSigner = new PrivateKeySigner({ privateJwk, keyId: 'did:jank:bob' });
+
+        const scope: PermissionScope = {
+          interface : DwnInterfaceName.Records,
+          method    : DwnMethodName.Write,
+          protocol  : 'https://example.com/',
```

Comment:
nit: keeps tests consistent and gives better assurance of normalization.

```suggestion
          protocol  : 'example.com/',
```

Reply:
@LiranCohen Thanks for the suggestion! Have made the change.